vue-components/src/components/TransformWidget.vue

```diff
@@ -0,0 +1,56 @@
+<script setup lang="ts">
+import type { ParameterValue, TransformDescription, TransformValue } from '../types'
+
+import ParamsWidget from './ParamsWidget.vue'
+
+interface Props {
+  pos: number
+  value: TransformValue
+  descriptions: { [name: string]: TransformDescription }
+}
+
+type Events = {
+  'update:type': [type: string]
+  'update:params': [params: { [name: string]: ParameterValue }]
+  remove: []
+}
+
+defineProps<Props>()
+const emit = defineEmits<Events>()
+</script>
+
+<template>
+  <div>
+    <q-select
+      filled
+      options-dense
+      stack-label
+      map-options
+      :model-value="value.name"
+      label="Transform"
```

Comment:
For slightly cleaner look, consiter dropping `label`

Reply:
Good idea, I like it